apisix/schema_def.lua

```diff
@@ -441,6 +443,13 @@ _M.route = {
         {required = {"upstream", "uris"}},
         {required = {"upstream_id", "uris"}},
         {required = {"service_id", "uris"}},
+        {required = {"script", "uri"}},
+        {required = {"script", "uris"}},
+    },
+    ["not"] = {
```

Comment:
that is the first time I know this way, it is cool ^_^

Reply:
but I do not find any test case for this case

Reply:
> but I do not find any test case for this case

`TEST 58` in t/admin/routes.t  is for this case  : )